### PR TITLE
With Xcode 5.2, the paths changed, this one has support for the new SDK

### PR DESCRIPTION
**Pull request by migueldeicaza**

### CODE REVIEW

**Comment by yyyyyyuanfei** [comment]
Xcode 5.2 ?!!  Isn't it 4.2?

**Comment by migueldeicaza** [comment]
Right, thinko.

File: Makefile [hunk 1]
```diff
@@ -6,10 +8,11 @@ demo.app: demo Info.plist
 	mkdir -p demo.app
 	cp demo demo.app/
 	cp Info.plist ResourceRules.plist demo.app/
-	codesign -f -s "iPhone Developer" --entitlements Entitlements.plist demo.app
+	codesign -f -s "iPhone Developer: Miguel de Icaza (QY8A78Y78J)" --entitlements Entitlements.plist demo.app
```
Comment (10p-freddo) [inline Makefile:11]: You've accidentally included your name here; could you remove it?

File: Makefile [hunk 2]
```diff
@@ -6,10 +8,11 @@ demo.app: demo Info.plist
...
 
 demo: demo.c
-	$(IOS_CC) -arch armv7 -isysroot /Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS5.0.sdk -framework CoreFoundation -o demo demo.c
+	$(IOS_CC) -arch armv7 -isysroot /Applications/Xcode.app//Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS5.0.sdk -framework CoreFoundation -o demo demo.c
+#/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS5.0.sdk -framework CoreFoundation -o demo demo.c
```
Comment (10p-freddo) [inline Makefile:15]: Could you also remove the 2 commented lines (other one is in Makefile)

**Comment by 10p-freddo** [comment]
Thanks for the pull request. Unfortunately, I'm not able to maintain the project any more. I'd encourage you to submit this request to one of the forks, or maintain your own fork. Cheers!

**Pull request closed without merging**